test/fuzz/config/corpus/corpus-0

```diff
@@ -0,0 +1 @@
+allocation_size|key_format=u,value_format=u,allocation_size=512,log=(enabled)
```

Comment:
Does the fuzzer allow for comments in the files? Probably with `#`? Whatever it is, I think a one-line comment about what is in this would be helpful.

Reply:
It doesn't allow comments since it can contain arbitrary binary data. This particular example gets talked about in the docs though.